cpp_linter/cli.py

```diff
@@ -69,6 +69,10 @@ class Args(UserDict):
     ignore_format: str = ""
     #: See :std:option:`--passive-reviews`.
     passive_reviews: bool = False
+    #: See :std:option:`--delete-review-comments`.
+    delete_comments: bool = True
```

Comment:
```suggestion
    delete_comments: bool = False
```

Because we changed the default below